logger/buffers/impl.go

```diff
@@ -0,0 +1,84 @@
+package buffers
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/ironsmile/nedomi/config"
+)
+
+// New returns a new Buffers logger.
+func New(cfg *config.LoggerSection) (*Buffers, error) {
+	b := &Buffers{
+		buffer: new(bytes.Buffer),
+	}
+	return b, nil
+}
+
+// Buffers writes all the logs to
+// bytes.Buffer for later examination
+type Buffers struct {
+	buffer *bytes.Buffer
```

Comment:
Primarily because this is the 6th implementation of this mock. I'm gonna make it an exported []string